setup.py

```diff
@@ -11,7 +11,12 @@
 import setuptools
 import torch
 import torch.utils.cpp_extension as torch_cpp_ext
```

Comment:
sadly most of these changes are caused by ruff, it apparently wasn't ruff formatted before.  Or my ruff is wrong?